packages/storybook/stories/components/Textarea.stories.tsx

```diff
@@ -0,0 +1,209 @@
+/* @license CC0-1.0 */
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Textarea } from '@utrecht/component-library-react/src/css-module/Textarea';
+import readme from '@utrecht/components/textarea/README.md?raw';
+import tokensDefinition from '@utrecht/components/textarea/tokens.json';
+import tokens from '@utrecht/design-tokens/dist/index.json';
+import React from 'react';
+import { designTokenStory } from './util';
+
+const arabicDecorator = (Story) => (
+  <div dir="rtl" lang="ar">
+    {Story()}
+  </div>
+);
+
+const meta = {
+  title: 'CSS Component/Textarea',
+  id: 'css-textarea',
+  component: Textarea,
+  argTypes: {
+    disabled: {
+      description: 'Disabled',
+      control: 'boolean',
+    },
+    invalid: {
+      description: 'Invalid',
+      control: 'boolean',
+    },
+    readOnly: {
+      description: 'Read-only',
+      control: 'boolean',
+    },
+    required: {
+      description: 'Required',
+      control: 'boolean',
+    },
+    focus: {
+      description: 'Focus',
+      control: 'boolean',
+    },
+    focusVisible: {
+      description: 'focusVisible',
+      control: 'boolean',
+    },
+    spellCheck: {
+      description: 'Spellcheck',
+      control: { type: 'select' },
+      options: ['', 'false', 'true'],
+    },
+  },
+  args: {
+    dir: 'auto',
+    disabled: false,
+    focus: false,
+    focusVisible: false,
+    hover: false,
```

Comment:
`focus`, `focusVisible` and `hover` are all broken in the Args Table:

<img width="1022" alt="Screenshot 2023-08-31 at 08 22 08" src="https://github.com/nl-design-system/utrecht/assets/30694/7b2af974-8bbb-4528-a941-9caf312852b4">

You now implemented them as separate render functions, only available to each individiual Focus and Hover story. When you implement them as a component, like we did in the MarkStory, they also work in the args table, and in the "Show code" example the class name will show up too.

https://github.com/nl-design-system/utrecht/blob/main/packages/storybook/stories/components/Mark.stories.tsx#L13-L29